requirements.in

```diff
@@ -10,4 +10,5 @@ py-githooks
 pytest
 semver
 twine
+typing_extensions
```

Comment:
Can't wait till we eliminate the duplication

Reply:
When we switch to yarn